repo_policy_compliance/blueprint.py

```diff
@@ -292,6 +290,17 @@ def health() -> Response:
     return Response(status=http.HTTPStatus.NO_CONTENT)
 
 
+@repo_policy_compliance.route(AUTH_HEALTH_ENDPOINT, methods=["GET"])
+@auth.login_required(role=RUNNER_ROLE)
+def auth_health() -> Response:
+    """Health check for authenticated requests.
+
+    Returns:
+        204 response otherwise.
+    """
+    return Response(status=http.HTTPStatus.NO_CONTENT)
```

Comment:
Shouldn't we check that the DB connection is working properly there ? Unless the connection is effectively executed before we reach the endpoint

Reply:
The DB connection is checked because the endpoint requires authentication with `RUNNER_ROLE`, which effectively calls `database.check_token` before reaching the endpoint.